src/gobapi/graphql_streaming/graphql2sql/graphql2sql.py

```diff
@@ -186,7 +186,8 @@ def _reset(self):
         self.relation_info = {}
 
     def _get_active(self, tablename: str):
-        return f"{tablename}.{FIELD.EXPIRATION_DATE} IS NULL OR {tablename}.{FIELD.EXPIRATION_DATE} > NOW()"
+        return f"({tablename}.{FIELD.EXPIRATION_DATE} IS NULL OR {tablename}.{FIELD.EXPIRATION_DATE} > NOW())" \
+               f"AND {tablename}.{FIELD.DATE_DELETED} IS NULL"
```

Comment:
Why not make a is_active() method that is called by all four methods

Reply:
_get_active is now being used on all four methods